app-vite/templates/bex/default/manifest-v2/background.js

```diff
@@ -11,20 +11,25 @@ chrome.browserAction.onClicked.addListener((/* tab */) => {
 })
 
 export default bexBackground((bridge /* , allActiveConnections */) => {
+  bridge.on('log', ({ data, respond }) => {
+    console.log(`[BEX] ${data.message}`, ...(data.data || []))
+    respond()
+  })
+
+  bridge.on('getTime', ({ respond }) => {
+    respond(Date.now())
+  })
+
   bridge.on('storage.get', ({ data, respond }) => {
-    if (data.key === null) {
+    const { key } = data
+    if (key === null) {
       chrome.storage.local.get(null, items => {
-        const result = []
-
-        // Group the items up into an array to take advantage of the bridge's chunk splitting.
-        for (const itemKey in items) {
-          result.push(items[itemKey])
-        }
-        respond(result)
+        // Group the values up into an array to take advantage of the bridge's chunk splitting.
+        respond(Object.values(items))
```

Comment:
Shouldn't this be done on manifes-v3 js version too?

Reply:
Good catch! Thanks.